test/WebJobs.Script.Tests/FileTraceWriterTests.cs

```diff
@@ -116,6 +116,30 @@ public void SetNewLogFile_EmptyDirectory_Succeeds()
             Assert.Equal(1, files.Length);
         }
 
+        [Fact]
+        public void WriteLogs_ReusesLastFile()
+        {
+            DirectoryInfo directory = new DirectoryInfo(_logFilePath);
+            directory.Create();
+
+            int count = directory.EnumerateFiles().Count();
+            Assert.Equal(0, count);
+
+            for (int i = 0; i < 3; i++)
+            {
+                FileTraceWriter traceWriter = new FileTraceWriter(_logFilePath, TraceLevel.Verbose);
+                traceWriter.Verbose("Testing");
+                traceWriter.Flush();
+            }
+
+            count = directory.EnumerateFiles().Count();
+            Assert.Equal(1, count);
```

Comment:
Before my fix, there would be 3 files - a new one was created for each new file logger